app-crypt/mcrypt/mcrypt-2.6.8.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Replacements for the old Unix crypt"
+DESCRIPTION="mcrypt is an enhanced replacement for the old crypt \
+command. mcrypt allows developers to use a wide range of encryption \
+functions without making drastic changes to their code. It contains the \
+actual encryption functions and provides a standardized mechanism for \
+accessing them."
+HOMEPAGE="http://mcrypt.sourceforge.net/"
+COPYRIGHT="1998-2007 Nikos Mavroyanopoulos"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/mcrypt/mcrypt-$portVersion.tar.gz"
+CHECKSUM_SHA256="5145aa844e54cca89ddab6fb7dd9e5952811d8d787c4f4bf27eb261e6c182098"
+SOURCE_DIR="mcrypt-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
Just being picky, x86 shouldn't be in here or commented as ?x86 as it builds on primary architectures x86_gcc2 and x86_64, other then that builds ok in x86_gcc2, nice work :+1:

Reply:
Yes, you are right, because we did not test it on x86.
But since the support for x86 primary arch has been dropped, I think we can change the meaning of the `?` for x86 primary arch:
* for recipes **without* support for x86 secondary arch, marking x86 primary arch as tested should now be understood, imo, as _"this was **not tested** on x86 but everything shows that it **would work** without any glitches."_
* for recipes **with** support for x86 secondary arch, marking x86 as tested should be understood as _"it **was tested** on x86 secondary arch and would therefore **also work** on x86 primary arch."_
* we would only mark x86 as untested for recipes which we think they might work but cannot assume it would.
* we would not mention x86 at all on recipes which do not support x86 secondary arch and for which we have no idea about the outcome on that arch.

I think this change would be quite fair.
Of course, this could be adopted for new recipes only. I'm not suggesting that we change the status of existing recipes.

Reply:
@fbrosson So should I always mark "x86" as untested or left it out from primary arch in all future recipes?

Reply:
If @Begasus agrees, I'd leave that as tested, otherwise you can either mark it as untested or remove it.
But I think it would be sad to not mention it.
And if we mention it, it would be sad to pretend we don't know if it works or not, because we all know it would perfectly work.

That said, some contributors could argue that x86 primary arch will disappear, so mentioning x86 in `ARCHITECTURES` is just noise, and they would be perfectly right.

So this is up to you. I'm not using x86 primary arch so any choice will be OK.

Reply:
Fair enough for me :)